bfcfg

```diff
@@ -664,6 +664,9 @@ sys_cfg_one_byte()
 #   SYS_ENABLE_I2C0      36            1
 #   SYS_DISABLE_FORCE_PXE_RETRY  37    1
 #   SYS_ENABLE_BMC_FIELD_MODE    38    1
+#   SYS_ENABLE_HEST      49            1
```

Comment:
We might need CeThreshold setting added here for ByteDance. Please see my comment on ezchip changes.

Reply:
Done